package.json

```diff
@@ -3,13 +3,28 @@
     "version": "2.3.1",
     "description": "Svelte plugin for prettier",
     "main": "plugin.js",
+    "module": "esm/plugin.mjs",
+    "exports": {
+        ".": {
+            "browser": {
+                "import": "./esm/standalone.mjs",
+                "default": "./standalone.js"
+            },
+            "default": "./plugin.js"
+        },
+        "./standalone": {
+            "import": "./esm/standalone.mjs",
+            "default": "./standalone.js"
+        }
+    },
     "files": [
         "plugin.js",
-        "plugin.js.map"
+        "plugin.js.map",
+        "esm/**/*.mjs",
+        "esm/**/*.mjs.map"
     ],
     "scripts": {
-        "build": "rollup -c",
-        "test": "ava",
```

Comment:
Any reason why you deleted the test script?

Reply:
Oh my bad, not sure how that happened sorry. I've amended the offending commit.